xcodecommand/fallbackRunner.go

```diff
@@ -0,0 +1,48 @@
+package xcodecommand
+
+import (
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/log"
+	version "github.com/hashicorp/go-version"
+)
+
+// FallbackRunner falls back to the raw xcodebuild runner in case xcbeautify (or other) is unavaialble
+type FallbackRunner struct {
+	runner         Runner
+	fallbackRunner Runner
+	logger         log.Logger
+}
+
+// NewFallbackRunner wraps an xcbeatufy (or other) runner
+func NewFallbackRunner(runner Runner, logger log.Logger, commandFactory command.Factory) *FallbackRunner {
+	return &FallbackRunner{
+		runner:         runner,
+		fallbackRunner: NewRawCommandRunner(logger, commandFactory),
+		logger:         logger,
+	}
+}
+
+// CheckInstall checks if the wrapped Runner is available,
+// if not changes the current runner to the raw xcodebuild runner
+func (f *FallbackRunner) CheckInstall() (*version.Version, error) {
+	if f.runner == nil || f.fallbackRunner == nil {
+		panic("runner or fallback runner is nil")
```

Comment:
Do we need this check? this never happens if the constructor (NewFallbackRunner) is used.
+ we don't do this check in the Run function.

If we want to not let consumers misuse FallbackRunner, we could make the struct private and the constructor could return a Runner interface.